pkg/bootstrap/testdata/all_golden.json

```diff
@@ -628,15 +628,15 @@
         }
       }
     }
-
+    
   ]
-
-
+  
+  
   ,
   "cluster_manager": {
     "outlier_detection": {
       "event_log_path": "/dev/stdout"
     }
   }
-
+  
```

Comment:
Nit: when reverting the golden files back, a number of the white space corrections get reverted as well. I know it's a little bit of a chore to remove.

Reply:
I just ran REFRESH_GOLDEN.. we shouldn't have manual changes here?